files/usr/bin/cinnamon-xlet-makepot

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/python3
+
+import os
+import json
+import subprocess
+import collections
+import argparse
+
+from gi.repository import GLib
+
+LOCALE_DIR = os.path.join(os.path.expanduser('~'), '.local/share/locale')
+
+USAGE_DESCRIPTION = """\
+Parses files in an applet, desklet or extension's directories, \
+extracts translatable strings, and then generates a ,pot file for them.
+"""
+
+USAGE_EPILOG = """\
+For example:
+cinnamon-xlet-makepot example@mydomain.org/
+
+Will generate a file called example@mydomain.org.pot, in the directory \
+example@mydomain.org/po/ which contains all of the strings that need to be \
+translated. Translators can then use this pot file to generate .po \
+files which contain translations for the extracted strings.
+"""
+
+try:
+    import polib
+except:
+    print("""
+
+    Module "polib" not available.
+
+    Please install the package "python-polib" and try again
```

Comment:
Hello, @collinss.

I have been "playing around" with the new `cinnamon-xlet-makepot` script and I found a couple of "inconsistencies".

Since the script now uses Python 3, the package to install now (in Debian based systems) is called `python3-polib`. Although, I thing that it should also be added to that message something like *or whatever name the package for the **polib** Python 3 module is called in your distribution*.

Reply:
Good point. I'll still include the package name for debian-based systems, but I'll change the wording and update the package name.